__tests__/commands.test.ts

```diff
@@ -22,20 +22,61 @@ callSerializer();
 
 describe('E2E', () => {
   describe('lint', () => {
+    const excludeFolders = ['arazzo-type-extensions-with-plugin'];
```

Comment:
Could you enlist all excluded folders (arazzo-valid-test-description, arazzo-not-valid-test-description)?

Reply:
They are skipped due to missing redocly.yaml, but I can add it here as well